src/mixins/labels.js

```diff
@@ -12,6 +11,8 @@ export const LabelsMixin = (superclass) =>
     class extends superclass {
         #texturesCache = {};
 
+        #labels = [];
```

Comment:
I think we should create a `labelsGroup` that will contain the labels for all atoms in the corresponding order as the atoms themselves